packages/runtime/datastore-definitions/src/test/types/validateDatastoreDefinitionsPrevious.generated.ts

```diff
@@ -245,7 +245,7 @@ use_old_InterfaceDeclaration_IFluidDataStoreRuntimeEvents(
 declare function get_old_TypeAliasDeclaration_Jsonable():
     TypeOnly<old.Jsonable>;
 declare function use_current_TypeAliasDeclaration_Jsonable(
-    use: TypeOnly<current.Jsonable>): void;
+    use: TypeOnly<current.Jsonable<any,any>>): void;
```

Comment:
breaking changes cannot be made in main

Reply:
Yes, I am not clear on the process with the changes being made. @tylerbutler, `next` is still alive? One more use for Beta 2 at least?

Reply:
Looks like plenty of changes still happening in `next`. Now I just wish it wasn't days behind so I could retarget more easily.

Reply:
Looks like `Jsonable` and `Serializable` have been tagged `internal` (impressive containment). Does that mean this could actually go to `main` directly?